src/utils.js

```diff
@@ -0,0 +1,3 @@
+export const extend = (a, b) => {
+  return Object.assign({}, a, b);
+};
```

Comment:
есть смысл использовать простую композицию. Если бы нужно было расширить какой-то уже существующий обьект, тогда - да.
```suggestion
export const extend = (a, b) => ({a, b});
```

Reply:
Как-то не поняла эту конструкцию сразу, попробовала её в коде. Результат не аналогичный, приложение прилегло. На проекте просто нельзя использовать spread-оператор, поэтому нам дали вот такую вспомогательную функцию. Её смысл я поняла: создать из нескольких объектов, которые пришли в аргументах, один новый.
Если я правильно поняла, предложенный код создает объект с объектами.
Коммитить предложенное не стала, так как не очень поняла идею. Можно чуть подробней?

Reply:
Прошу прощения, мое упущение: 
`(a,b) => ({...a, ...b})`

Reply:
Но в принципе, это дело вкуса и ничего против Object.assign я не имею